test/utilities/genetics-utils.js

```diff
@@ -69,6 +69,67 @@ describe("GeneticsUtils.convertDashAllelesObjectToABAlleles()", function() {
   });
 });
 
+describe("GeneticsUtils.alleleStringContainsAlleles()", function() {
+
```

Comment:
I'm not seeing a case for if a null is passed in here - the other tests here do cover that.